core/storage/job/gae_models.py

```diff
@@ -94,17 +110,38 @@ def get_recent_jobs(cls, limit, recency_msec):
 
     @classmethod
     def get_all_unfinished_jobs(cls, limit):
+        """Get all jobs that are unfinished.
```

Comment:
If there's a limit, should "all" be struck? So, something like "Gets a list of unfinished jobs."